### PR TITLE
Receive create user and store postgres

### PR DESCRIPTION
**Pull request by mpkagel**

### CODE REVIEW

File: src/Housing.Selection.Context/Selection/MappingProfile.cs [hunk 1]
```diff
@@ -73,4 +90,4 @@ public MappingProfile()
                 .ForMember(d => d.batch, o => o.MapFrom(s => s.Batch));
         }
     }
-}
+}
```
Comment (stormrubio3000) [inline src/Housing.Selection.Context/Selection/MappingProfile.cs]: Need to add another line so that the file ends in an empty line for Lenting

File: src/Housing.Selection.Context/ServiceBus/ServiceMessaging.cs [hunk 2]
```diff
@@ -33,13 +33,13 @@ protected ServiceMessaging(IQueueFactory queueFactory, string sendQ, string rece
...
             SendQ = _queueFactory.GetQueueClient(sendQ);
-            ReceiveQ = _queueFactory.GetQueueClient(receiveQ);
-            ReceiveQ.RegisterMessageHandler(HandleMessage,
-                 exceptionevent =>
-                 {
-                     Logger.LogDebug("Something went wrong");
-                     throw exceptionevent.Exception;
-                 });
+            //ReceiveQ = _queueFactory.GetQueueClient(receiveQ);
+            //ReceiveQ.RegisterMessageHandler(HandleMessage,
```
Comment (stormrubio3000) [inline src/Housing.Selection.Context/ServiceBus/ServiceMessaging.cs]: This needs to be deleted or kept in but not commented out.

File: src/Housing.Selection.Context/ServiceBus/ServiceMessaging.cs [hunk 3]
```diff
@@ -48,15 +48,15 @@ protected ServiceMessaging(IQueueFactory queueFactory, string sendQ, string rece
...
-        {
-            if (nextId != null && nextMessage == null && message.CorrelationId == nextId)
-            {
-                nextMessage = message;
-                await ReceiveQ.CompleteAsync(message.SystemProperties.LockToken);
-                nextId = null;
-            }
-        }
+        //private async Task HandleMessage(Message message, System.Threading.CancellationToken token)
+        //{
```
Comment (stormrubio3000) [inline src/Housing.Selection.Context/ServiceBus/ServiceMessaging.cs]: same for here

File: src/Housing.Selection.Library/HousingModels/User.cs [hunk 4]
```diff
@@ -30,7 +30,7 @@ public class User
         public string Type { get; set; }
 
         /// <value>Batch user is assigned to.</value>
-        [Required]
+        //[Required]
```
Comment (stormrubio3000) [inline src/Housing.Selection.Library/HousingModels/User.cs]: delete or keep

File: src/Housing.Selection.Service/appsettings.json [hunk 5]
```diff
@@ -4,10 +4,13 @@
     "DbInDockerMachineContainer": "Server=192.168.99.100;Database=HousingSelectionDb;User Id=postgres;"
   },
   "Settings": {
-    "Queues": false,
+    "Queues": true,
     "DbConnection": "DbInDockerStack",
     "ServiceHubConnection": "DockerNetworkHttp"
   },
+  "Queues": {
+    "Queue": "mattq"
```
Comment (stormrubio3000) [inline src/Housing.Selection.Service/appsettings.json]: is this the right queue name for deployment?

File: src/Housing.Selection.Library/HousingModels/User.cs (hunk 4 above)
Comment (mpkagel) [reply src/Housing.Selection.Library/HousingModels/User.cs]: I am removing this. I am also documenting right here that batch id has been removed as a requirement for users in the Selection service. So users are not required to be assigned a batch anymore, but they will typically be assigned a batch.

File: src/Housing.Selection.Service/appsettings.json (hunk 5 above)
Comment (mpkagel) [reply src/Housing.Selection.Service/appsettings.json]: No, and I am changing this.

File: src/Housing.Selection.Service/Program.cs [hunk 6]
```diff
@@ -22,7 +22,7 @@ public static void Main(string[] args)
             Log.Logger = new LoggerConfiguration()
                 .Enrich.FromLogContext()
                 .WriteTo.Console()
-                .WriteTo.File("log.log", rollingInterval: RollingInterval.Day)
+                //.WriteTo.File("log.log", rollingInterval: RollingInterval.Day)
```
Comment (stormrubio3000) [inline src/Housing.Selection.Service/Program.cs]: Should be deleted so it doesn't come into deployment

File: src/Housing.Selection.Service/Housing.Selection.Service.csproj [hunk 7]
```diff
@@ -28,4 +28,6 @@
     <ProjectReference Include="..\Housing.Selection.Library\Housing.Selection.Library.csproj" />
   </ItemGroup>
 
+  <ProjectExtensions><VisualStudio><UserProperties appsettings_1Development_1json__JSONSchema="" /></VisualStudio></ProjectExtensions>
```
Comment (stormrubio3000) [inline src/Housing.Selection.Service/Housing.Selection.Service.csproj]: Why is this added?
